dbt_subprojects/daily_spellbook/models/aave/base/aave_v3_base_interest_rates.sql

```diff
@@ -0,0 +1,28 @@
+{{ config(
+   schema = 'aave_v3_base'
+  , materialized = 'incremental'
+  , file_format = 'delta'
+  , incremental_strategy = 'merge'
+  , unique_key = ['reserve', 'symbol', 'hour']
+  , alias = 'interest'
+  , post_hook='{{ expose_spells(blockchains = \'["base"]\',
+                                spell_type = "project",
+                                spell_name = "aave_v3",
+                                contributors = \'["mikeghen1","batwayne", "chuxin"]\') }}'
+  )
+}}
+
+select 
+  a.reserve, 
+  t.symbol,
+  date_trunc('hour',a.evt_block_time) as hour, 
+  avg(CAST(a.liquidityRate AS DOUBLE)) / 1e27 as deposit_apy, 
+  avg(CAST(a.stableBorrowRate AS DOUBLE)) / 1e27 as stable_borrow_apy, 
+  avg(CAST(a.variableBorrowRate AS DOUBLE)) / 1e27 as variable_borrow_apy
+from {{ source('aave_v3_base', 'Pool_evt_ReserveDataUpdated') }} a
+left join {{ source('tokens_base', 'erc20') }} t
+on a.reserve = t.contract_address
```

Comment:
`symbol` is currently null.

Reply:
I changed to use `tokens.erc20`: https://github.com/duneanalytics/spellbook/pull/7404/commits/ceb747b6922de227703938a8fe5a0c9aca2f9dee